guides/source/configuring.md

```diff
@@ -1416,7 +1416,7 @@ Below is a comprehensive list of all the initializers found in Rails in the orde
 Database pooling
 ----------------
 
-Active Record database connections are managed by `ActiveRecord::ConnectionAdapters::ConnectionPool` which ensures that a connection pool synchronizes the amount of thread access to a limited number of database connections. This limit defaults to 5 and can be configured in `database.yml`.
+Active Record database connections are managed by `ActiveRecord::ConnectionAdapters::ConnectionPool` which ensures that a connection pool synchronizes the amount of thread access to a limited number of database connections. This limit defaults to `5` and can be configured in `database.yml`.
```

Comment:
"5" is valid Ruby code, but there is no need to highlight it as a code block for me.